par/core.py

```diff
@@ -61,7 +62,7 @@ def _update_repo_sessions(sessions: Dict[str, Any]):
 
 
 # Session operations - simplified from actions.py
-def start_session(label: str):
+def start_session(label: str, open_session: bool = False):
```

Comment:
Are these changes from a different PR https://github.com/coplane/par/pull/8 ? Looks like you may have checked out a new branch from that branch instead of main?